src/main/webapp/app/shared/metis/postings-markdown-editor/postings-markdown-editor.component.ts

```diff
@@ -44,13 +44,30 @@ export class PostingsMarkdownEditorComponent implements OnInit, ControlValueAcce
     }
 
     /**
-     * writes the current value of a form group and propagates the change
+     * this lifecycle hook is required to avoid causing "Expression has changed after it was checked"-error when dismissing all changes in the markdown editor
+     * on dismissing the edit-create-modal -> we do not want to store changes in the create-edit-modal that are not saved
+     */
+    ngAfterContentChecked() {
+        this.cdref.detectChanges();
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    _onChange = (val: string) => {};
```

Comment:
What is this expression/function needed for? I see it assigns a string to it later on, but I am not quite sure the usage of it.

Reply:
Ok I see 👍  Thanks for documenting it a bit, this will make it clearer for future developers to understand what's happening